jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/ContainerParameters.java

```diff
@@ -42,7 +41,7 @@
   private ImageFormat format = ImageFormat.Docker;
   private List<String> ports = Collections.emptyList();
   private Map<String, String> labels = Collections.emptyMap();
-  private String appRoot = JavaLayerConfigurations.DEFAULT_APP_ROOT;
+  private String appRoot = "";
```

Comment:
I'm thinking this may affect Maven. Gradle now sets this value [below](https://github.com/GoogleContainerTools/jib/pull/923/files#diff-aef8bf567c91df6622aa0ea29ed0c682R136), but I don't see Maven doing a similar setup in this PR. Need to confirm if Maven works without any problem.

Reply:
I think there is no side effect.
In this inner class `com.google.cloud.tools.jib.maven.JibPluginConfiguration.ContainerParameters`, there is still this code : 
```
@Parameter private String appRoot = JavaLayerConfigurations.DEFAULT_APP_ROOT;
```

Reply:
Oops, you're right. The file is not shared, but exclusive to the Gradle plugin.